mix.exs

```diff
@@ -66,6 +66,6 @@ defmodule Guardian.DB.Mixfile do
   end
 
   defp aliases do
-    [test: ["ecto.create --quiet", "ecto.migrate", "test"]]
+    [test: ["ecto.drop --quiet", "ecto.create --quiet", "test"]]
```

Comment:
- Why do you want to drop and then create the database every time?
- Why did you remove the migration?